.all-contributorsrc

```diff
@@ -1420,6 +1420,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "gshpychka",
+      "name": "Glib Shpychka",
+      "avatar_url": "https://avatars.githubusercontent.com/u/23005347?v=4",
+      "profile": "https://github.com/gshpychka",
+      "contributions": [
+        "bug"
+      ]
```

Comment:
0.02$: This change seems unrelated to the fix

Reply:
This is the record for the individual who's reported this bug... Need I submit 2 separate PRs?